README.md

```diff
@@ -0,0 +1,11 @@
+Version 2 of Dragon Age 2 themed Tic Tac Toe. This time, I built the game using Ruby, jQuery, Sinatra, and JSON.
+
+### To Play:
+
+Clone repo: git@github.com:shanemachine92/web_TTT.git
+
+cd into main directory, JSON_MvT
+
+run 'ruby lib/app.rb'
```

Comment:
forgot the bundle step,
I would include the git clone command,
the bundle is missing pry, which `app.rb` requires,
the views don't load by default `Errno::ENOENT at /
No such file or directory @ rb_sysopen - /web_TTT/lib/views/index.erb`